# 四足机器狗的落脚点坐标，各数组长度为3
float32[] footend_FL  # 左前腿
float32[] footend_FR  # 右前腿
float32[] footend_BL  # 左后腿
float32[] footend_BR  # 右后腿